app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -1,29 +1,85 @@
 <template>
   <div>
-    <v-form>
-      <v-row>
-        <v-col cols="12" xl="4">
-          <v-text-field dense flat label="Name" data-test="text-formName" v-model="formName" />
-        </v-col>
-        <v-col cols="12" xl="8">
-          <v-text-field
-            dense
-            flat
-            label="Description"
-            data-test="text-formDescription"
-            v-model="formDescription"
-          />
-        </v-col>
-      </v-row>
-      <v-btn color="primary" @click="submitFormSchema">
-        <span>Save Form Design</span>
-      </v-btn>
-    </v-form>
+    <v-card class="mb-5">
+      <v-toolbar flat color="grey lighten-3">
+        <v-card-title>Form Settings</v-card-title>
+      </v-toolbar>
+      <v-card-text>
+        <v-form ref="form" v-model="valid" lazy-validation>
+          <v-container>
+            <v-row>
+              <v-col cols="12" xl="4">
+                <v-text-field
+                  dense
+                  flat
+                  label="Name"
+                  data-test="text-formName"
+                  v-model="formName"
+                  :rules="formNameRules"
+                />
+              </v-col>
+              <v-col cols="12" xl="8">
+                <v-text-field
+                  dense
+                  flat
+                  label="Description"
+                  data-test="text-formDescription"
+                  v-model="formDescription"
+                  :rules="formDescriptionRules"
```

Comment:
Description isn't mandatory in the db, should it be?

Reply:
It was failing to POST with a 422 without it, just made it mandatory for demo

Reply:
ahh, it's the schema validation rule... 
models , line 79 should have ```{ type: ['string', 'null'], maxLength: 255 }``` to allow nulls or empty strings and enforce the length limit.

Reply:
Ok thanks, will add that in my next PR and make the field optional on the FE too